test/Microsoft.DotNet.ApiCompatibility.Tests/Rules/AttributesMustMatchTests.cs

```diff
@@ -20,17 +20,14 @@ public class AttributesMustMatchTests
          * - ReturnValues
          * - Constructors
          * - Generic Parameters
-         * 
+         *
          * Grouped into:
          * - Type
          * - Member
          */
 
         private static readonly TestRuleFactory s_ruleFactory = new((settings, context) => new AttributesMustMatch(settings, context));
 
-        private static ISymbolFilter GetAccessibilityAndAttributeSymbolFiltersAsComposite(params string[] excludeAttributeFiles) =>
-            new CompositeSymbolFilter().Add(new AccessibilitySymbolFilter(false)).Add(new DocIdSymbolFilter(excludeAttributeFiles));
```

Comment:
I should probably review these changes in VS to better understand them. Before, it was clear what the filter that we were passing in here was: `CompositeSymbolFilter` with an `AccessibilitySymbolFilter` and a `DocIdSymbolFilter`.

I feel like that information gets lost now with the replaced call to `SymbolFilterFactory.GetFilterFromFiles([filePath], respectInternals: false);`. Maybe that's just me not understand the change sufficiently.

Reply:
I added a new optional argument to the symbol filter factory methods to take an accessibiilty filter if needed.